elements/pl-matrix-output/pl-matrix-output.py

```diff
@@ -6,19 +6,37 @@
 
 def prepare(element_html, data):
     element = lxml.html.fragment_fromstring(element_html)
-    pl.check_attribs(element, required_attribs=[], optional_attribs=['digits'])
+    required_attribs = []
+    optional_attribs = ['digits', 'display-matlab-tab', 'display-mathematica-tab', 'display-python-tab']
+    pl.check_attribs(element, required_attribs, optional_attribs)
 
 
 def render(element_html, data):
     element = lxml.html.fragment_fromstring(element_html)
     digits = pl.get_integer_attrib(element, 'digits', 2)
+    display_matlab_tab = pl.get_boolean_attrib(element, 'display-matlab-tab', True)
+    display_mathematica_tab = pl.get_boolean_attrib(element, 'display-mathematica-tab', False)
+    display_python_tab = pl.get_boolean_attrib(element, 'display-python-tab', True)
+
+    # Default tab is Matlab
+    active_tab_matlab = True
+    active_tab_mathematica = False
+    active_tab_python = False
+    # If Matlab Display is False, will cycle through to next displayed tab
+    if display_matlab_tab is False:
+        active_tab_matlab = False
+        if display_mathematica_tab is True:
+            active_tab_mathematica = True
+        elif display_python_tab is True:
+            active_tab_python = True
 
     matlab_data = ''
+    mathematica_data = ''
     python_data = 'import numpy as np\n\n'
     for child in element:
         if child.tag == 'variable':
             # Raise exception of variable does not have a name
-            pl.check_attribs(child, required_attribs=['params-name'], optional_attribs=[])
+            pl.check_attribs(child, required_attribs=['params-name'], optional_attribs=['params-comment', 'params-digits'])
```

Comment:
Ah, that makes sense. Any reason to rename digits since it is also an attribute of the element?

Reply:
No, `digits` is fine. We generally prefer short name. The only reason for `params-name` has `params` on it is for consistency with other elements. We often want to specify variables from inside `params`, `correct-answers`, etc, so we try and name these `params-name`, `correct-answers-name`, etc, so that we are very clear about which structure the name should be used in.